projects/ngrx.io/content/guide/eslint-plugin/rules/signal-state-no-arrays-at-root-level.md

```diff
@@ -10,3 +10,33 @@ signalState should accept a record or dictionary as an input argument.
 
 <!-- Everything above this generated, do not edit -->
 <!-- MANUAL-DOC:START -->
+
+## Rule Details
+
+This rule ensure that a Signal State shouldn't accept an array type at root level.
+
+Examples of **correct** code for this rule:
```

Comment:
It would be great to add an example that uses an array here - as this will probably be the fix in most cases.

```ts
const store = withState({ arrayAsProperty: ['foo', 'bar'] })
```

Reply:
Thanks for the feedback, indeed when I used withState on my project signalStores I already faced this problem and fixed it this way, forgot to put it as example.